published/009-2016-March-28.md

```diff
@@ -0,0 +1,157 @@
+
+Many people jumped to IPFS as a possible solution to this problem. With a permanent filesystem, unpublishing wouldn't be possible. Here's one post titled [How to use IPFS to fix npm](http://amreldib.com/blog/HowToUseIpfsToFixNpm); here's [an issue](https://github.com/ipmjs/ipmjs/issues/9) on a new GitHub organization, ipmjs, trying to find consensus on how to fix npm using a permanent storage system; here's an npm module, [cowpen](https://www.npmjs.com/package/cowpen) that publishes modules directly to IPFS; here's [another decentralized package manager](https://github.com/mhhf/spore) that sprung up using IPFS and Ethereum.
+
+The IPFS community has been thinking about immutable package managers for a long time. @daviddias has a project called `registry-mirror`, which gets npm packages by using IPFS. He's written about a presentation he gave for it, [here](http://blog.daviddias.me/2015/12/08/stellar-module-management); the [source code](https://github.com/diasdavid/registry-mirror) is here.
+
+On a similar note, [gx](//github.com/whyrusleeping/gx), a package manager for Go, was also mentioned in a lot of the discussions about npm and package managers, especially on Hackernews. In the past two weeks, the project has recieved over 1000 stars - before, there were around 50.
+
+The discussion about how to best use IPFS as a package manager is ongoing. Jump on GitHub if you have something to say; we're listening.
+
+### DNS outage
+
+We're using DigitalOcean for providing ipfs.io DNS. On Tuesday DigitalOcean DNS was hit by a severe outage lasting hours, which took the public gateway at ipfs.io down. We switched to DNSimple in an ad-hoc fashion and brought ipfs.io back while DigitalOcean was still down, but this incident obviously hit us on the wrong foot a bit. We'll be working to never get taken down this way again. Here's a few things we'll do:
+
+- codified DNS zones, and tools to upload them to DNS providers
+- one or two backup DNS providers
+- monitoring and failover procedures
+
+We'll post a more detailed postmorten on our blog in the next few days.
+
+### Orbit
+
+@haadcode has been working on improvements to orbit-db, ipfs-log and Orbit. The message history fetching is
+now more stable and the UI feedback for loading messages is fixed. All this work will improve the user experience of Orbit.
+
+### js-ipfs-init
+
+`js-ipfs init` works! @noffle finished the remaining pieces this week, including CLI usage. This included a handful of auxiliary PRs that cascaded out of that work. 
```

Comment:
https://github.com/ipfs/js-ipfs/pull/91 and https://github.com/ipfs/js-ipfs/pull/96

It may be informative to note that this makes js-ipfs' init process produce IPFS repos that are compatible with go-ipfs' -- this is big!